SingularityBase/src/main/java/com/hubspot/singularity/SingularityDeleteResult.java

```diff
@@ -1,5 +1,5 @@
 package com.hubspot.singularity;
 
 public enum SingularityDeleteResult {
-  DELETED, DIDNT_EXIST;
+  DELETING, DELETED, DIDNT_EXIST;
```

Comment:
added this state to better reflect the state of the delete when it's first requested